auditlog_security/models/auditlog_rule.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2021 Therp B.V.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api, modules, _
+
+
+class AuditlogRule(models.Model):
+    _inherit = 'auditlog.rule'
+
+    field_ids = fields.Many2many(
+            'ir.model.fields',
+            required=True,)
```

Comment:
It can't be required always, because we need to keep supporting pre-existing rules that are just there but don't (need to) have our new cool fields functionality. So I say we remove the requiredness here, and make it required only in the form view and only when a checkbox is enabled saying that for this rule, we want to use the "Publicly visible log lines" subfunctionality.

Reply:
I hadn't thought about that , you are correct. we will still have "normal" auditlog.rule.